api/v1/views/index.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3
+"""
+Views' index file
+"""
+
+from flask import jsonify
+from api.v1.views import app_views
+
+
+@app_views.route('/status', methods=['GET'], strict_slashes=False)
+def jsonify_app():
+    """ Function that returns a JSON """
+    return jsonify({"status": "OK"})
```

Comment:
Can we make this print a new line at the end as well?

Reply:
I have added this config line `app.config['JSONIFY_PRETTYPRINT_REGULAR'] = True` in the app.py which adds a newline at the end and makes the output look nicer.

Reply:
On my side when I curl status it prints the output but without the new line. Perhaps its something on my end

Reply:
vagrant@vagrant-ubuntu-trusty-64:~/AirBnB_clone_v3$ curl -X GET http://0.0.0.0:5000/api/v1/status
{
  "status": "OK"
}vagrant@vagrant-ubuntu-trusty-64:~/AirBnB_clone_v3$


This is how the output looks on my side